src/pages/home/report/ReportActionCompose.js

```diff
@@ -207,7 +209,7 @@ class ReportActionCompose extends React.Component {
                         )}
                     </AttachmentModal>
                     <TouchableOpacity
-                        style={[styles.chatItemSubmitButton, styles.buttonSuccess]}
+                        style={[styles.chatItemSubmitButton, styles.buttonSuccess, {backgroundColor: this.state.submitButtoncolor}]}
```

Comment:
Inline styles should be avoided. We should create a new style for the different states rather than overwrite the existing style set in `styles.buttonSuccess`.